### ISSUE TITLE
distutils deprecation warning

### ISSUE THREAD

**Issue by grawlinson** [fsfe/reuse-tool#450]
You're probably already aware of this, but it's nice to have an issue that can be pointed at.

When running `python setup.py`, this warning pops up:

```
reuse-tool-0.14.0/setup.py:11: DeprecationWarning:
  The distutils package is deprecated and slated for removal in Python 3.12.
  Use setuptools or check PEP 632 for potential alternatives.
```

No hurry to get this one fixed, 3.12 is a long way off. :)

**Comment by ferdnyc** [comment]
> No hurry to get this one fixed, 3.12 is a long way off. :)

...Was that meant to be ironic? [The new 12-month release schedule](https://discuss.python.org/t/accepted-pep-602-annual-release-cycle-for-python/2296) means that, with Python 3.10 already out, Python 3.11 is expected to release in October 2022. Some time _very_ shortly after that, 3.12 development will begin. (Just like, right now, the Python development tree is already on version 3.11a3.)

Basically, in less than a year there will be people running [pre-release] Python 3.12.

Given how little time that is (and how many people think it's a lot farther away than it really is), I wouldn't be surprised to see the distutils removal pushed back until Python 3.13 or even later. Because if it _isn't_ pushed back, it's happening a lot sooner than people realize!

Luckily, for this repo it's a trivial fix that just requires switching from `distutils`  to `setuptools` for the required class. I've opened #451 to take care of that.

**Comment by grawlinson** [comment]
No irony was intended.

A fix can be created before October 2022, which **is** a long way off. And you’ve just provided a fix, so thank you for that! 🙂
